pkg/search/postgres/query/numeric_query.go

```diff
@@ -110,13 +110,24 @@ func getEquivalentGoFuncForNumericQuery(prefix string, value float64) func(found
 	}
 }
 
-func createNumericQuery(root string, prefix string, value float64) WhereClause {
-	var valueStr string
-	if _, fraction := math.Modf(value); fraction > 0 {
-		valueStr = fmt.Sprintf("%0.2f", value)
-	} else {
-		valueStr = fmt.Sprintf("%0.0f", value)
+func createNumericRangeQuery(root string, lower float64, upper float64) WhereClause {
+	lowerStr := readable.Float(lower, 2)
+	upperStr := readable.Float(upper, 2)
+	return WhereClause{
+		Query:  fmt.Sprintf("(%s > $$) AND (%s < $$)", root, root),
```

Comment:
Why are these exclusive? That seems unintuitive

Reply:
My thoughts on this: This is really a preference thing and there is no clear answer. Having said that I am for keeping the range exclusive. Since otherwise I have to specify 31-59 if it is inclusive and I want >30 < 60. Which is rather unintuitive IMO. If someone wants it inclusive, = 30, >30 < 60, = 60 as a union should give them everything that they need!

Reply:
So I think either we just make the format capable of understanding `[]` `()` or we just allow multiple values in the same chip so we can do >=30 <=60 if you want inclusivity.

Reply:
The problem is multiple values in the same chip mean "OR" right now. So if you do `>=30,<=60`, it becomes ">=30 OR <= 60" which is... everything.

In any case, we don't have that many numeric fields, this is mostly a concern for days. There, the inclusivity versus exclusivity thing is kind of moot anyway because the query goes down to a specific number of seconds.

Reply:
It would have to be in the same chip and be parsed out

Reply:
Hmm, so do you think we should replace the whole syntax from `a-b` to `<a>b` and have that implicitly mean `AND`?

Reply:
It's a bit ugly but that would be unambiguous

Reply:
Follow up question, should we even expose this to users in this form? Like wouldn't this just be for internal use? Where did this use case come from?

Reply:
This is an internal use-case (for the dashboards and such). I don't see a reason to go out of our way to make it impossible for users to use, but we also don't have to advertise it or anything

Reply:
Another follow up, if it's for dashboards, then we should try and figure out how to expose this as part of a query language aggregation?

Reply:
I think those are all good for follow-up work, but those will need more careful design and thought. This was meant to just be a quicker fix to unblock this use case within the existing query langauge.

Reply:
Perf impact seems negligible, can you create some tickets for the follow ups?

Reply:
Done: https://issues.redhat.com/browse/ROX-11528